scripts/esbuild-shims/dirname.js

```diff
@@ -0,0 +1,7 @@
+import { dirname } from 'node:path'
```

Comment:
Could you please create a `README.md` file in this directory explaining what all these shims are for?